index.html

```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </ul>
+
+            <ul class="tech2">
+              <li class="Language1">
+                <img src="Rectangle2.png" alt="Rectangle" /> Frameworks
+              </li>
+              <ul class="TechStacks">
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+              </ul>
+            </ul>
+
+            <ul class="tech">
+              <li class="Language1">
+                <img src="Rectangle3.png" alt="Rectangle" /> skills
+              </li>
+              <ul class="TechStacks">
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Developement</li>
+              </ul>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- html form -->
+
+    <section class="Contact" id="Contact">
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+          
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+
+              <li class="newInputName01">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="First name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+              <li class="newInputName02">
+                <input
+                  name="LastName"
+                  type="Text"
+                  id="Name"
+                  placeholder="Last name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps11"></div>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  maxlength="30"
+                  required
+                />
```

Comment:
- [ ] You added a `maxlength` attribute to the email input, this is not part of the requirement, your email is not meant to have a `maxlength`, I'll suggest you remove this. :+1: :100: